classic-cli/Format-Cpp.ps1

```
@@ -0,0 +1,56 @@
+
+$formatExtensions = [System.Collections.Generic.HashSet[string]]::new([System.StringComparer]::OrdinalIgnoreCase)
+foreach ($extension in @('.cc', '.cpp', '.h', '.hpp')) {
+    [void] $formatExtensions.Add($extension)
+}
+
+$excludedDirectoryNames = [System.Collections.Generic.HashSet[string]]::new([System.StringComparer]::OrdinalIgnoreCase)
+foreach ($directoryName in @('TES5Edit', 'tests', 'build', 'CMakeFiles', 'Testing', 'vcpkg_installed')) {
+    [void] $excludedDirectoryNames.Add($directoryName)
+}
+
+function Get-FormatCppFiles {
```

PSScriptAnalyzer (1.25.0)

[warning] The cmdlet 'Get-FormatCppFiles' uses a plural noun. A singular noun should be used instead.

Suggested fix: Singularized correction of 'Get-FormatCppFiles'

(PSUseSingularNouns)